test/test_torch.py

```diff
@@ -11251,9 +11247,18 @@ def test_bitwise_ops(self):
         x_clone ^= y
         self.assertEqual(x_clone, xor_result)
 
-    def test_invert(self):
-        x = torch.ByteTensor([0, 1, 1])
-        self.assertEqual((~x).tolist(), [1, 0, 0])
+    def test_op_invert(self):
+        for t in (torch.uint8, torch.int8, torch.int16, torch.int32, torch.int64):
+            a = torch.arange(127, dtype=t)
+            self.assertEqual(~a, a.bitwise_not())
+        a = torch.BoolTensor([True, False])
```

Comment:
Use `torch.tensor([True, False])` (dtype is inferred)

We are avoiding the classes like torch.BoolTensor and torch.FloatTensor in favor of dtypes